test_spider.py

```diff
@@ -0,0 +1,19 @@
+import unittest
```

Comment:
A very basic offline test of the ThoroughSpider.parse method. Scrapy comes with a more sophisticated integration test feature called Spider Contracts (http://doc.scrapy.org/en/latest/topics/contracts.html) but I could not get it to work with the custom command line arguments I added to my spider. Their documentation is a little spare, I'm assuming it's not a fully-featured part of Scrapy yet.